runtime/interpreter/value.go

```diff
@@ -19517,7 +19536,16 @@ type SomeStorable struct {
 	Storable atree.Storable
 }
 
-var _ atree.Storable = SomeStorable{}
+var _ atree.ContainerStorable = SomeStorable{}
+
+func (s SomeStorable) HasPointer() bool {
```

Comment:
What is the purpose of `Storable.HasPointer`?

Reply:
`ContainerStorable.HasPointer` is used to set a flag bit during encoding if encoded slab refers to other slabs.

`SomeStorable.Storable` might contain reference to another slab.

Reply:
Thanks for the explanation! 

Could you please add a comment/docstring to the function on the atree side? See https://github.com/onflow/atree/pull/352#discussion_r1366192016